sbt

```diff
@@ -269,15 +283,37 @@ download_url () {
 }
 
 acquire_sbt_jar () {
+
+  # if none of the options touched coursier_launcher_version, use the coursier
```

Comment:
Doesn't look like any options touch it.  So maybe the env var should just set the version.